etl/controller/metrics.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/block-explorer/blob/master/LICENSE.md.
+
+package controller
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/insolar/block-explorer/instrumentation/metrics"
+)
+
+var (
+	DataQueue = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
rename DataQueue to be more clearly, for instance, IncompletePulsesQueue

Reply:
porefachil metrics